dangerzone/gui/main_window.py

```diff
@@ -78,6 +78,47 @@
 handle it manually.</p>
 """
 
+WSL_NEEDS_INSTALL_MSG = """\
+<p>Dangerzone requires a Microsoft Windows feature called
+<a href="https://learn.microsoft.com/en-us/windows/wsl/">
+Windows Subsystem for Linux (WSL)
+</a>, which is not present in your system.
+<p>You can let Dangerzone download and install it now, or quit Dangerzone and install it
+on your own.</p>
+"""
+
+WSL_NEEDS_REBOOT_MSG = """\
+<p>Dangerzone has installed the Windows Subsystem for Linux (WSL), but you need to
+reboot your computer before Dangerzone can use it.</p>
+<p>You can either reboot your computer now, or quit Dangerzone and reboot it
+later.</p>
+"""
```

Comment:
Same, plus I removed the last sentence as it's a duplicate with the options shown to the user.
```suggestion
WSL_NEEDS_REBOOT_MSG = """\
<p>The Windows Subsystem for Linux (WSL) is now installed, but you need to reboot your computer
to make Dangerzone work.</p>
"""
```

Reply:
Hm, I want to explain to the user that rebooting is the only option to move forward, but they can do it later on, if they want to. I fear that "Quit Dangerzone" on its own does not get this message across. So, how about this rewording:

```suggestion
WSL_NEEDS_REBOOT_MSG = """\
<p>The Windows Subsystem for Linux (WSL) is now installed, but you need to
reboot your computer to make Dangerzone work.</p>
<p>You can either reboot your computer now, or quit Dangerzone and reboot it
later.</p>
"""
```